### ISSUE TITLE
Run CI on our own systems

### ISSUE THREAD

**Issue by Munksgaard** [diku-dk/futhark-benchmarks#35]
Now that we can [actually test futhark-benchmarks on our own machines](https://github.com/diku-dk/futhark/blob/master/.github/workflows/main.yml#L333-L355), perhaps we should do so in this repo as well?

**Comment by athas** [comment]
Yes, sure.  We just need to figure out how to make the runners listen to this repository as well.  Fortunately, we have hired a mercenary to do things like that.

**Comment by WilliamDue** [comment]
I think [runner groups](https://docs.github.com/en/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups) could be used for this.

**Comment by Munksgaard** [comment]
> I think [runner groups](https://docs.github.com/en/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups) could be used for this.

That link is down, I think the new link is [this one?](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/managing-access-to-self-hosted-runners-using-groups)

**Comment by WilliamDue** [comment]
> > I think [runner groups](https://docs.github.com/en/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups) could be used for this.
> 
> That link is down, I think the new link is [this one?](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/managing-access-to-self-hosted-runners-using-groups)

Yes this looks correct.
